.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ executors:
     parameters:
       image:
         type: string
-        default: "10.13"
+        default: "10.19"
```

Comment:
I needed to bump the image because 10.13 was using a *very* old Yarn version (1.10, we're now at 1.22) which didn't have full yarn-path support. Another option would be to have a specific executor for this test, but I'm not sure it's worth it, your call 🤔

Reply:
This is a bit muddy for me. Would this mean all tests would run in 10.19 after this? Because it looks to me that way.

Since we minver at 10.13, the first LTS minor, I'm pretty sure we want to keep testing against that. But perhaps this is not going to affect other tests. I think it'd be fine if this only affected the yarn e2e tests (provided it would indeed work for 10.13 otherwise).

Reply:
No it currently affects the other tests as - makes sense you'd prefer to keep it to 10.13. I've added an executor that only targets the PnP test.